Source/Provers/SMTLib/ProverInterface.cs

```diff
@@ -382,7 +488,9 @@ private void FlushAxioms()
       TypeDecls.Clear();
       foreach (string s in Axioms) {
         Contract.Assert(s != null);
-        if (s != "true")
+	if (s.StartsWith("(define-fun"))
+	  SendCommon(s);
+        else if (s != "true")
           SendCommon("(assert " + s + ")");
```

Comment:
Use braces around the `if` alternatives.

Reply:
Fixed